actionview/lib/action_view/helpers/asset_tag_helper.rb

```diff
@@ -373,7 +374,7 @@ def preload_link_tag(source, options = {})
         preload_link += "; integrity=#{integrity}" if integrity
         preload_link += "; nopush" if nopush
 
-        send_preload_links_header([preload_link])
+        send_preload_links_header([preload_link]) if send_link_header
```

Comment:
Should the `preload_link` construction above should be included in the `if` as well?

Reply:
Yeah, that makes sense!